src/xbot/common/command/BaseCommandGroup.java

```diff
@@ -1,16 +1,37 @@
 package xbot.common.command;
 
+import org.apache.log4j.Logger;
+
 import com.google.inject.Inject;
 
 import edu.wpi.first.wpilibj.command.CommandGroup;
 
 public class BaseCommandGroup extends CommandGroup {
+    
+    protected Logger log;
+    
     @Inject
     SmartDashboardCommandPutter commandPutter;
     
+    public BaseCommandGroup() {
+        log = Logger.getLogger(this.getName());
+    }
+
+    public BaseCommandGroup(String name) {
+        super(name);
+        log = Logger.getLogger(this.getName());
```

Comment:
I find this much easier to follow if you use the local `name`; otherwise I have to search up the tree to confirm that `super` eventually sets a variable which will allow this to work properly.